### PR TITLE
Wrap configureRuns() transaction

### PR DESCRIPTION
**Pull request by evmiguel**

### CODE REVIEW

File: client/components/TestQueue/index.jsx [hunk 1]
```diff
@@ -142,7 +142,13 @@ class TestQueue extends Component {
         const atBrowserRunSets = [];
```
Comment (evmiguel) [inline client/components/TestQueue/index.jsx:142]: This came in with lint fixes.

File: server/services/RunService.js [hunk 2]
```diff
@@ -105,17 +105,24 @@ const db = require('../models/index');
...
-    test_version_id,
-    apg_example_ids,
-    at_browser_pairs
-}) {
+async function configureRuns(
+    { test_version_id, apg_example_ids, at_browser_pairs },
+    commit = true
+) {
     try {
+        await db.sequelize.query('BEGIN;');
```
Comment (evmiguel) [inline server/services/RunService.js:113]: This was the only way I could create a transaction that was compatible with the `ROLLBACK` in `dbCleaner`

Comment (spectranaut) [reply server/services/RunService.js:113]: Unless we want to change the way we do dbCleaner, I think we might as well run with this -- but I agree it's unsatisfying!

**Comment by rmeritz** [comment]
I think this is a legit solution! LGTM.

**Pull request merged**